<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GimbalOption</class>
 <widget class="QWidget" name="GimbalOption">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="3">
    <spacer name="horizontalSpacer_7">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>36</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <widget class="QGroupBox" name="groupBox_Gimbal_IP_Option">
     <property name="title">
      <string>云台IP设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="2">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1">
       <widget class="QGroupBox" name="groupBox_1_changeOption">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>100</height>
         </size>
        </property>
        <property name="title">
         <string>更改设置：</string>
        </property>
        <widget class="QLabel" name="label_tcp_device_1_1">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>TCP端口：</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_tcpiplabel_device_1_1">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>50</y>
           <width>21</width>
           <height>16</height>
          </rect>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>IP：</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_tcpportlabel_device_1_1">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>50</y>
           <width>41</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>port：</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_tcpip_device_1_1">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>50</y>
           <width>141</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_tcpport_device_1_1">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>50</y>
           <width>51</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_tcpsave_device_1_1">
         <property name="geometry">
          <rect>
           <x>340</x>
           <y>50</y>
           <width>71</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="1">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>100</horstretch>
          <verstretch>100</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>100</height>
         </size>
        </property>
        <property name="title">
         <string>参数查询：</string>
        </property>
        <widget class="QLabel" name="label_tcp_device_1">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>TCP端口：</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_tcpiplabel_device_1">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>50</y>
           <width>21</width>
           <height>16</height>
          </rect>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>IP：</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_tcpportlabel_device_1">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>50</y>
           <width>41</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>port：</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_tcpip_device_1">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>50</y>
           <width>141</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_tcpport_device_1">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>50</y>
           <width>51</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_tcpconnect_device_1">
         <property name="geometry">
          <rect>
           <x>340</x>
           <y>50</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>连接</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer_9">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>94</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QGroupBox" name="groupBox_Gimbal_RealtimeAckhaul_Option">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>200</height>
      </size>
     </property>
     <property name="title">
      <string>角度回传设置</string>
     </property>
     <widget class="QCheckBox" name="checkBoxRealtimeAckhaulOpenSet">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>281</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>角度回传功能(重启生效)</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_RealtimeAckhaul">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>30</y>
        <width>100</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRealtimeAckhaulClose">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>70</y>
        <width>131</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string>关闭实时回传</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRealtimeAckhaulOpen">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>131</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string>打开实时回传</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_RealtimeAckhaul">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>30</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>回传频率(hz):</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer_10">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <spacer name="horizontalSpacer_12">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>32</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2">
    <widget class="QGroupBox" name="groupBox_Gimbal_Speed_Option">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>70</height>
      </size>
     </property>
     <property name="title">
      <string>俯仰速度设置</string>
     </property>
     <widget class="QSpinBox" name="spinBox_speed_horizontal">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_speed_horrizontal">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>水平速度：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_speed_vertical">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>30</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>垂直速度：</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_speed_vertical">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>20</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
     </widget>
    </widget>
   </item>
   <item row="6" column="2">
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>25</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="3">
    <spacer name="horizontalSpacer_11">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>87</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="title">
      <string>一键归零</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_13">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>253</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnOneClickZeroing">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>一键归零</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_14">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>252</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="2">
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>26</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer_8">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>87</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
